src/main/java/thePackmaster/powers/calamitypack/GrowingFlamesPower.java

```diff
@@ -0,0 +1,35 @@
+package thePackmaster.powers.calamitypack;
+
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.localization.PowerStrings;
+import thePackmaster.SpireAnniversary5Mod;
+import thePackmaster.powers.AbstractPackmasterPower;
+import thePackmaster.powers.shamanpack.IgnitePower;
+import thePackmaster.util.Wiz;
+
+public class GrowingFlamesPower extends AbstractPackmasterPower {
+    public static final String POWER_ID = SpireAnniversary5Mod.makeID("GrowingFlames");
+    private static final PowerStrings powerStrings = CardCrawlGame.languagePack.getPowerStrings(POWER_ID);
+    public static final String NAME = powerStrings.NAME;
+    public static final String[] DESCRIPTIONS = powerStrings.DESCRIPTIONS;
+
+    public GrowingFlamesPower(AbstractCreature owner, int amount) {
+        super(POWER_ID, NAME, PowerType.BUFF,false, owner, amount);
+    }
+
+    @Override
+    public void atStartOfTurnPostDraw() {
+        if (!AbstractDungeon.getMonsters().areMonstersBasicallyDead()) {
+            this.flash();
+            Wiz.forAllMonstersLiving(m -> this.addToBot(new ApplyPowerAction(m, this.owner, new IgnitePower(m, this.amount))));
```

Comment:
There is an AllEnemyApplyPowerAction, that might help here because this will be slow